_docs/tutorials/how-to-use-jwt-in-sn-client-js.md

```diff
@@ -30,16 +30,92 @@ In order to get JWT work you have to update your web.config, filling the *Symmet
 </sensenet>
 ```
 ```diff
-Please note that JWT Authentication works only with HTTPS connection. To continue with IIS Express, you can press F4 on the Web project to enable SSL and check the https URL.
+JWT Authentication is available only with HTTPS connection. To continue with IIS Express, you can press F4 on the Web project to enable SSL and check the **https URL**.
```

Comment:
JWT authentication is available only through HTTPS connection. To set it for IIS Express, press F4 on the web project to the Properties page and enable SSL. After that you can check the generated HTTPS URL.